README.md

```diff
@@ -8,9 +8,9 @@ To install the prerequisites:
 
 ``npm install``
 
-You also need the LAME, SoX, and curl packages. On Ubuntu/Debian, you can use the following command:
+You also need the LAME, SoX, curl and math.js packages. On Ubuntu/Debian, you can use the following command:
 
-``apt-get install lame sox curl``
+``apt-get install lame sox curl mathjs``
```

Comment:
I'm not sure that adding mathjs here is necessary. Typically for a node package you would use npm (or yarn or similar) to install the node module locally to the project in `node_modules` using `npm install`.